### PR TITLE
Add docker-compose file for local development

### PR DESCRIPTION
**Pull request by adamgraves-choices**

### CODE REVIEW

File: templates/c2s_local_dev/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# C2S docker-compose file for local development
+
+1. Clone this repo locally `git https://github.com/nhsuk/nhsuk-rancher-templates.git`
```
Comment (st3v3nhunt) [inline templates/c2s_local_dev/README.md:3]: You might want to add `clone` to that command e.g. `git clone https://github.com/nhsuk/nhsuk-rancher-templates.git`

File: templates/c2s_local_dev/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+2. Change working directory `cd nhsuk-rancher-templates/templates/c2s_local_dev/`
+3. Create a `.env` file with the required ENV variables (currently only SPLUNK_HEC_TOKEN). Useful ones to define are:
+    1. FE_TAG= defines which version of the 'connecting-to-services' app to deploy
+    2. API_TAG= defines which version of the 'nearby-services-api' app to deploy
+4. Run `docker-compose up --build --force-recreate`. This will bring up your "stack" with port 3000 exposed locally. The `--build` and `--force-recreate` will ensure you always start the stack from scratch.
```
Comment (st3v3nhunt) [inline templates/c2s_local_dev/README.md:8]: :+1: Good info

**Review by st3v3nhunt — CHANGES_REQUESTED** [review]
When running with default values either set in the shell or within a `.env` file I get the error:
`ERROR: Invalid interpolation format for "<section>"...`. The way the default values are being set just doesn't seem to want to work.

I've had a look online and i've not found anything that suggests it shouldn't work. However, how about having the `.env` set with the defaults rather than them being in `docker-compose.yml`?

**Comment by neilbmclaughlin** [comment]
No longer required - later branch includes these commits

**Pull request closed without merging**